langevals/evaluators/langevals/langevals_langevals/select_best_compare.py

```
@@ -79,6 +79,65 @@
+Pick the best of N candidate replies — there is no task description or
+reference answer, so compare them on their own merits.
+
+Candidates:
+{candidates}
+
+Look across the candidates and decide which one is the best reply.
+Briefly explain WHY it's better than the others, then pick the winning
+slot label. Use "tie" only when no candidate is clearly better.
+"""
+
+# Every shipped default judge prompt, one per (golden × input) presence combo.
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)